package.json

```diff
@@ -149,5 +149,6 @@
   "publishConfig": {
     "registry": "https://registry.npmjs.org"
   },
-  "types": "lib/index.d.ts"
+  "types": "lib/index.d.ts",
+  "packageManager": "yarn@1.22.22+sha1.ac34549e6aa8e7ead463a7407e1c7390f61a6610"
```

Comment:
could you remove the `packageManager` field?

Reply:
The `packageManager` was added by yarn automatically.
I removed this field.